README.rst

```diff
@@ -62,38 +62,47 @@ them, please visit the following URLs:
 
 -  Announcements: http://lists.idyll.org/listinfo/khmer-announce
 
-We chat at https://gitter.im/dib-lab/khmer and the maintainers can be
-contacted at khmer-project@idyll.org.
+We have a slack chat that [you can join](LINK/EMAIL/JOIN) and the maintainers
```

Comment:
I would prefer not to proliferate slack membership beyond people I know.
However, we could pay for slack and then set up a channel to whom we can invite
specific people.  But unfortunately you need an invitation to be sent out.
There are ways around that but it's all just very annoying.

I think we should

* primarily point people at khmer issue tracker
* advertise khmer@ mailing list